pkg/apis/ark/v1/backup.go

```diff
@@ -132,3 +136,7 @@ type BackupList struct {
 	metav1.ListMeta `json:"metadata"`
 	Items           []Backup `json:"items"`
 }
+
+func (b *Backup) NamespaceAndName() string {
```

Comment:
Do you want to add this to every custom type, or have a generic helper func that works on metav1.ObjectMeta?

Reply:
Yeah, the latter's better. Still getting used to go-style "generics"..